src/frontend/packages/lib_components/src/data/stores/useP2PVideoConfig/index.ts

```diff
@@ -0,0 +1,30 @@
+import { create } from 'zustand';
+
+interface P2PLiveConfigStore {
+  isP2PEnabled: boolean;
+  stunServersUrls: string[];
+  webTorrentServerTrackerUrls: string[];
+  setP2PLiveConfig: (
+    isP2PEnabled: boolean,
```

Comment:
This property is not accurate. In the back application, the setting name is `live_enabled`. So I understand the purpose of it is to enable the feature for live only. Here the setting name let me think the purpose is for both live and vod